apps/mark-scan/backend/src/globals.ts

```diff
@@ -31,3 +32,11 @@ export const MARK_SCAN_WORKSPACE =
   (NODE_ENV === 'development'
     ? join(__dirname, '../dev-workspace')
     : undefined);
+
+/**
+ * What kind of hardware is the app running on?
+ */
+export const MARK_SCAN_BMD_MODEL: BmdModelNumber = process.env
```

Comment:
The feature flag module isn't initialized at this point so I check the environment variable directly. Not sure if this is the best way to do it, but seemed relatively harmless since the flag library and environment variables are very closely coupled.

Reply:
Could replace with a `getMarkScanBmdModel()` function instead if there are no constraints WRT this being a constant.